src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/formatWriter/JsonRecordWriterProvider.java

```diff
@@ -48,12 +51,14 @@ public void write(SinkRecord record) {
                     try {
                         Object value = record.value();
                         if (value instanceof Struct) {
-                            byte[] rawJson = converter.fromConnectData(
-                                    record.topic(),
-                                    record.valueSchema(),
-                                    value);
-                            out.write(rawJson);
-                            out.write(LINE_SEPARATOR_BYTES);
+                            byte[] rawJson = converter.fromConnectData(record.topic(),record.valueSchema(),value);
+                            if(ArrayUtils.isEmpty(rawJson)) {

```

Comment:
im not sure we want to filter these - there might be a reason why someone wants to ingest an empty array